Tests/Specs/unified-test-format/tests/valid-pass/poc-retryable-writes.json

```diff
@@ -297,13 +297,15 @@
             "ordered": true
           },
           "expectResult": {
-            "insertedCount": {
-              "$$unsetOrMatches": 2
-            },
-            "insertedIds": {
-              "$$unsetOrMatches": {
-                "0": 3,
-                "1": 4
+            "$$unsetOrMatches": {
```

Comment:
this seems to be a change that was made for CSHARP purposes that never had a corresponding DRIVERS ticket for everyone to sync
(https://github.com/mongodb/specifications/commit/ec947b3700c38c2de80d6dff79921416983200fe)